lib/facility.rb

```diff
@@ -1,14 +1,59 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, :address, :phone, :services, :collected_fees, :registered_vehicles, :registration_date
```

Comment:
We do not want or need the `:registration_date` in the attr_reader here because we don't have an attribute called `@registration_date` that we need to read.